domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -30,7 +45,7 @@
   private static final String RESPOND_404_WITH_TEXT_PLAIN       = "respond404WithTextPlain";
   private static final String RESPOND_500_WITH_TEXT_PLAIN       = "respond500WithTextPlain";
   private static final String NOT_FOUND = "Not found";
-
+  
```

Comment:
Can you configure your editor/IDE to use the `.editorconfig` https://dev.folio.org/guidelines/contributing/#coding-style ? This helps to reduce whitespace changes (pull requests, code review, …).
Please remove the whitespace at this line.

Reply:
Still working on setting up editorconfig for eclipse.  I removed the whitespace by hand for now.